kuka_kr3_support/urdf/kr3r540.xacro

```diff
@@ -3,11 +3,36 @@
 
   <xacro:arg name="prefix" default="" />
   <xacro:arg name="use_mock_hardware" default="false" />
-  
+  <xacro:arg name="mock_sensor_commands" default="false" />
+  <xacro:arg name="sim_gazebo_classic" default="false" />
+  <xacro:arg name="sim_gazebo" default="false" />
+  <xacro:arg name="simulation_controllers" default="" />
+
+   <!-- create link fixed to the "world" -->
+   <!-- using-external-joints-and-links approach -->
+  <link name="world" />
+  <joint name="$(arg prefix)base_joint" type="fixed">
+    <origin xyz="0 0 0" rpy="0 0 0" />          <!-- position robot in the world -->    
+    <parent link="world" />
+    <child link="$(arg prefix)base_link" />
+  </joint>  
+
```

Comment:
```suggestion

  <gazebo reference="world">
  </gazebo>

```

Reply:
yes, set it here as "base_link" I just showed you the place with this comment. Its different place in the structure that can make difference.

Reply:
Ok. i already tried that and what i observed is robot model not grounded, and just rolling off on the plane randomly.
i think we need world to base_link joint for sure as mentioned in the gazebo docs for robot model to be fixed on the ground in the first place.